docker/Dockerfile

```
@@ -38,13 +40,26 @@ RUN WHL=$(ls -d /wheelhouse/*) && \
 
+COPY --from=rust-toolchain /usr/local/cargo /usr/local/cargo
+COPY --from=rust-toolchain /usr/local/rustup /usr/local/rustup
+ENV CARGO_HOME=/usr/local/cargo
+ENV RUSTUP_HOME=/usr/local/rustup
+ENV PATH="${CARGO_HOME}/bin:${PATH}"
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends ca-certificates gcc libc6-dev && \
+    rm -rf /var/lib/apt/lists/*
+
 COPY --from=build /workspace/dist /wheelhouse
 RUN WHL=$(ls -d /wheelhouse/*) && \
```

Hadolint (2.14.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)